index.html

```diff
@@ -0,0 +1,396 @@
+            <!--    End schedule       -->
+            <!--    Start Main Carousel      -->
+        <div id="carousel-example-generic" class="carousel slide main-carousel" data-ride="carousel">
+          <!-- Wrapper for slides -->
+          <div class="carousel-inner" role="listbox">
+            <div class="item active">
+              <img src="images/news/29252139.jpg" alt="messi">
+              <div class="carousel-caption main-carousel-caption hidden-xs">
+                <h1>FC Barcelona Get Straight Sack To Training</h1>
+              </div>
+            </div>
+            <div class="item">
+              <img src="images/news/29508058.jpg" alt="enrique">
+              <div class="carousel-caption main-carousel-caption hidden-xs">
+                <h1>Top Ten Stories Of The Week</h1>
+              </div>
+            </div>
+            <div class="item">
+              <img src="images/news/33518581.jpg" alt="suarez">
+              <div class="carousel-caption main-carousel-caption hidden-xs">
+                <h1>Luis Suárez receives 2015/16 Golden Shoe Award</h1>
+              </div>
+            </div>
+            <div class="item">
+              <img src="images/news/33816026.jpg" alt="doro">
+              <div class="carousel-caption main-carousel-caption hidden-xs">
+                <h1>Barça quartet up for Ballon d’Or award</h1>
+              </div>
+            </div>
+          </div>
+            <!-- Controls -->
+          <a class="left carousel-control hidden-lg hidden-md hidden-sm" href="#carousel-example-generic" role="button" data-slide="prev">
+            <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+            <span class="sr-only">Previous</span>
+          </a>
+          <a class="right carousel-control hidden-lg hidden-md hidden-sm" href="#carousel-example-generic" role="button" data-slide="next">
+            <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+            <span class="sr-only">Next</span>
+          </a>
+          <ol class="carousel-indicators hidden-xs">
+      		<li data-target="#carousel-example-generic" data-slide-to="0" class="active"><img src="images/news/29252139.jpg" alt="messi"></li>
+            <li data-target="#carousel-example-generic" data-slide-to="1"><img src="images/news/29508058.jpg" alt="enrique"></li>
+            <li data-target="#carousel-example-generic" data-slide-to="2"><img src="images/news/33518581.jpg" alt="suarez"></li>
+            <li data-target="#carousel-example-generic" data-slide-to="3"><img src="images/news/33816026.jpg" alt="doro"></li>
+          </ol>
+        </div>
+            <!--    End Main Carousel        -->
+            <!--    Start Section about -->
+        <section class="about text-center">
+            <div class="container">
+                <h2>CAMP NOU EXPERIENCE</h2>
+                <p class="lead">History happens here. Live it with the Tour & Museum.</p>
```

Comment:
`&` should be written as `&amp;` in HTML, that's why GitHub highlights it in Red.